Assets/Controllers/GameManager.cs

```diff
@@ -24,18 +26,32 @@ public void StartPressed()
         hexMap = FindObjectOfType<HexMap>();
         pawnKeeper = FindObjectOfType<PawnKeeper>();
         players = new List<Player>();
-        string newPlayerName = "Syd";
-        Player newPlayer = new Player(newPlayerName);
-        players.Add(newPlayer);
-        activePlayer = newPlayer;
+        for (int i = 0; i < startingNumberOfPlayers; i++)
```

Comment:
What is the purpose of the hard coded "Syd_" for the PlayerName? Also this could be pulled out into another method that returns a newly populated players list. It would be both easier to test and a bit cleaner to read.